csv/readcsv.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	runtime "github.com/ahsanfayaz52/EmployeeManagementSystem"
+	"github.com/ahsanfayaz52/EmployeeManagementSystem/models"
+)
+
+func main() {
```

Comment:
all main files should be in `cmd`... this could be `cmd/importer/main.go`

Reply:
done